### PR TITLE
[TD-13418]<fix>taosBenchmark create database config

### PR DESCRIPTION
**Pull request by zhaoyanggh**

### CODE REVIEW

File: src/benchOutput.c [hunk 1]
```diff
@@ -84,51 +84,51 @@ void printfInsertMetaToFileStream(FILE *fp) {
...
             fprintf(fp, "  days:                  \033[33m%d\033[0m\n",
                     g_arguments->db[i].dbCfg.days);
         }
-        if (g_arguments->db[i].dbCfg.keep > 0) {
+        if (g_arguments->db[i].dbCfg.keep >= 0) {
             fprintf(fp, "  keep:                  \033[33m%d\033[0m\n",
                     g_arguments->db[i].dbCfg.keep);
         }
-        if (g_arguments->db[i].dbCfg.replica > 0) {
+        if (g_arguments->db[i].dbCfg.replica >= 0) {
```
Comment (sangshuduo) [inline src/benchOutput.c:103]: replica can not be 0 as design

File: src/benchOutput.c [hunk 2]
```diff
@@ -84,51 +84,51 @@ void printfInsertMetaToFileStream(FILE *fp) {
...
             fprintf(fp, "  walLevel:              \033[33m%d\033[0m\n",
                     g_arguments->db[i].dbCfg.walLevel);
         }
-        if (g_arguments->db[i].dbCfg.fsync > 0) {
+        if (g_arguments->db[i].dbCfg.fsync >= 0) {
             fprintf(fp, "  fsync:                 \033[33m%d\033[0m\n",
                     g_arguments->db[i].dbCfg.fsync);
         }
-        if (g_arguments->db[i].dbCfg.quorum > 0) {
+        if (g_arguments->db[i].dbCfg.quorum >= 0) {
```
Comment (sangshuduo) [inline src/benchOutput.c:131]: quorum cannot be 0 as design

**Review by sangshuduo — COMMENTED** [review]
should we prompt the user if the value is out of scope we support?

**Comment by zhaoyanggh** [comment]
> should we prompt the user if the value is out of scope we support?

I want to make it dynamic and correspond with taos client and display the error message from taos client if the value is out of scope

**Review by afwerar — APPROVED** [review]
LGTM

File: src/benchJsonOpt.c [hunk 3]
```diff
@@ -303,7 +303,20 @@ static int getMetaFromInsertJsonFile(cJSON *json) {
...
     for (int i = 0; i < dbSize; ++i) {
         SDataBase *database = &(g_arguments->db[i]);
-        cJSON *    dbinfos = cJSON_GetArrayItem(dbs, i);
+        database->dbCfg.minRows = -1;
+        database->dbCfg.maxRows = -1;
+        database->dbCfg.comp = -1;
+        database->dbCfg.walLevel = -1;
+        database->dbCfg.cacheLast = -1;
+        database->dbCfg.fsync = -1;
+        database->dbCfg.replica = -1;
```
Comment (sangshuduo) [inline src/benchJsonOpt.c:312]: then if the user did not specify replica in JSON file, we will pick the default value -1 to create a database?

Comment (zhaoyanggh) [reply src/benchJsonOpt.c:312]: we only set the database parameter when its value is larger than or equal to zero, set -1 is default and do nothing when create database

**Review by sangshuduo — APPROVED** [review]
LGTM

**Pull request merged**